app/assets/stylesheets/map.scss

```
@@ -0,0 +1,173 @@
+}
+
+// Overrides for zoom controls to make them match GOVUK buttons
+// https://design-system.service.gov.uk/components/button/
+//
+// GOVUK buttons have an invisible outline, used for high contrast modes. Our buttons are cropped by
+// their container so we use their border instead.
+//
+// Also introduces a thick black divider between the buttons, made with a pseudo element, appearing
+// on focus. Together with the yellow background, this copies the 2-colour style GOVUK Frontend
+// buttons and links have.
+.leaflet-bar {
+  border-radius: 0; // Remove rounded corners
+  background: #CCCCCC;
+
+  @media (forced-colors: active) {
+    background: canvasText;
+  }
+
+  a {
+
+    // Outlines don't work because of the buttons being so close together so turn off
+    outline: none;
+
+    &:first-child {
+      border-bottom: none;
+      margin-bottom: 1px;
+    }
+
+    &:last-child {
+      // Allow it to contain the absolutely positioned divider we show between buttons
+      // when one is focused
+      position: relative;
+    }
+
+    // Hover style is darker background
+    &:hover {
+      background-color: $zoom-button-hover-colour;
+    }
+
+    // Styles that apply if focused with or without the :hover or :active state
+    &:focus {
+      // Button focus is shown by a change in background colour.
+      // When colours are overridden, for example when users have Windows high
+      // contrast mode on, backgrounds disappear, so we need to ensure there's a
+      // visible outline so it can be set to a colour by the OS.
+      border: solid 2px $govuk-focus-colour;
+      box-sizing: border-box; // make sure height includes the border
+      line-height: 22px; // subtract the border from the height (normally 26px)
+
+      // The inline display box for the button text overlaps the button edge.
+      // This is only visible in high contrast mode because of the background colour being set.
+      @media (forced-colors: active) {
+        overflow: hidden;
+      }
+    }
+
+    // Focused style sets the button to the focus colour
+    &:focus:not(:active):not(:hover) {
+      color: $govuk-focus-text-colour;
+      background-color: $govuk-focus-colour;
+    }
+
+    // Shared styles for the divider between buttons that appears when either are focused
+    &:last-child:focus:not(:active):not(:hover):before,
+    &:first-child:focus:not(:active):not(:hover) + a:before {
+      content: "";
+      position: absolute;
+      height: 2px;
+      background: $govuk-focus-text-colour;
+
+      // hide in forced-color mode, where the normal focus state the divider is part of does not show
+      @media (forced-colors: active) {
+        display: none;
+      }
+    }
+
+    // Styles for the divider, specifc to the second button being focused
+    &:last-child:focus:not(:active):not(:hover):before {
+      left: -2px; // Subtract 2px left border
+      top: -3px; // Subtract 2px top border and position so it overlaps space above button
```

```suggestion
      top: -3px; // Shift divider up 1px so it overlaps buttons equally, and another 2px to compensate for the reduces height of the last button (due to the border)
```

If you reverse the order of these two blocks you could omit the first part of the sentence here to save space, if we assume people are reading top-to-bottom.